test/val/lib_runtime_shlax7.c

```diff
@@ -0,0 +1,44 @@
+/*
+  !!DESCRIPTION!! A small test for shlax7.
+  !!ORIGIN!!
+  !!LICENCE!!
+  !!AUTHOR!!
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <limits.h>
+#include <errno.h>
+
+int main (void)
+{
+    unsigned int ai = 0, ti, refi;
+    unsigned char ac = 0, tc, refc;
+
```

Comment:
This seems broken - how can the test yield useful results when the bits are all 0 ?

Reply:
The idea was to test every possible number, but of course I forgot the ++ :sweat_smile: